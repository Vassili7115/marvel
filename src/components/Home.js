import React from 'react';

import '../stylesheets/home.scss';

const Home = () => (
  <div>
    <div className="container">
      <div className="item" />
      <div className="item" />
      <div className="item" />
      <div className="item" />
      <div className="item">
        <div className="text-box">
          <a href="/characters">
            <span className="text">DISCOVER MARVEL</span>
          </a>
        </div>
      </div>
      <div className="item" />
      <div className="item" />
      <div className="item" />
    </div>

  </div>
);

export default Home;
