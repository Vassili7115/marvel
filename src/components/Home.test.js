import React from 'react';
import { mount, configure } from 'enzyme';
import toJson from 'enzyme-to-json';

import Adapter from 'enzyme-adapter-react-16';

import Home from './Home';

configure({ adapter: new Adapter() });

describe('Home component', () => {
  it('Home snapshots', () => {
    const wrapper = mount(
      <Home />,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
