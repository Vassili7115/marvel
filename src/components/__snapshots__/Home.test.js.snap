// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component Home snapshots 1`] = `
<Home>
  <div>
    <div
      className="container"
    >
      <div
        className="item"
      />
      <div
        className="item"
      />
      <div
        className="item"
      />
      <div
        className="item"
      />
      <div
        className="item"
      >
        <div
          className="text-box"
        >
          <a
            href="/characters"
          >
            <span
              className="text"
            >
              DISCOVER MARVEL
            </span>
          </a>
        </div>
      </div>
      <div
        className="item"
      />
      <div
        className="item"
      />
      <div
        className="item"
      />
    </div>
  </div>
</Home>
`;
